cost-analyzer/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
-appVersion: "1.92.0"
+appVersion: "1.97.0"
 description: A Helm chart that sets up Kubecost, Prometheus, and Grafana to monitor
   cloud costs.
 name: cost-analyzer
-version: "1.92.0"
+version: "1.97.0"
```

Comment:
Please update to `1.98.0` as this is the current version of the `develop` branch.